momentum/4/modules/msg-gen.md

```diff
@@ -154,6 +154,32 @@ Maximum allowed time in the future for which a transmission can be scheduled. Ti
 
 </dl>
 
+### <a name="modules.msg_gen.scheduled_send"></a> Scheduled Send
+
+As of Momentum 4.4, a message can be scheduled to be sent at some time in the future as part of the `msg_gen` module. Scheduled send currently only applies to REST injections.
+
+The old version of this functionality was known as scheduled generation, and it was retired in Momentum 4.2.28.
+Scheduled send differs from scheduled generation as it is simpler, requires less configuration, and saves disk space.
+The old functionality would place a generated message on disk until it was time to send, but the new functionality instead utilizes the systems memory.
```

Comment:
Can simplify this:

differs from scheduled generation in that the message is now immediately generated, but queued in Momentum's delayed queue until the scheduled start time.